.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Any particular reason to switch to JS config format? JSON currently has an advantage that editors can suggest the properties and check values if using Prettier extension. If using JS format it's probably worth adding the following comments to preserve suggestions, but still feels unnecessary, as it doesn't really change anything (?).

```suggestion
// @ts-check
/** @type {import("prettier").Options} */
module.exports = {
```

Reply:
For ESLint it's because in the future the JS format will be [updated and preferred](https://eslint.org/docs/latest/use/configure/configuration-files-new)

For Prettier there was no reason

For Nuxt - TypeScript is the new default, and additionally it'd be best to use TypeScript everywhere

Reply:
Actually, as I am now checking it, the new ESLint config can already be used without an env var

Reply:
Yeah other stuff is understandable and nice, that's why I asked under Prettier file change <img src="https://cdn.discordapp.com/emojis/991452305683652669.webp?size=48&quality=lossless" width="22" alt=":ioa:" />

Reply:
I've reverted the Prettier file and added a type annotation for ESLint, just in case :)